apis/fluent-en_GB/extensions/jdk8/atrium-api-fluent-en_GB-jdk8-jvm/src/module/module-info.java

```diff
@@ -1,10 +1,10 @@
-module ch.tutteli.atrium.api.fluent.en_GB.jdk8 {
-    requires ch.tutteli.atrium.api.fluent.en_GB;
-    requires ch.tutteli.atrium.domain.builders;
-    requires ch.tutteli.kbox;
-    requires ch.tutteli.niok;
-    requires kotlin.stdlib;
-    requires java.base;
+// module ch.tutteli.atrium.api.fluent.en_GB.jdk8 {
```

Comment:
keep, you basically don't need to modify the jdk8 extension, only deprecate the methods. We want to keep the extension to stay backward compatible. We will remove everything with 1.0.0. The deprecation messages are i.a. here to help users migrate from the jdk8 extension to the normal jvm module

Reply:
the build fails with an error:
```
package is empty or does not exist: ch.tutteli.atrium.api.fluent.en_GB.jdk8        
    exports ch.tutteli.atrium.api.fluent.en_GB.jdk8
```
Should it remain commented?

Reply:
will change once you copy the file from the jdk8 extension to the normal jvm module instead of moving.

Reply:
Maybe we can remove the whole folder apis\fluent-en_GB\extensions\jdk8\atrium-api-fluent-en_GB-jdk8-jvm
what do you think?

Reply:
no, we need it for backward compatibility

Reply:
got it